addons/web_editor/static/src/js/wysiwyg/widgets/link_dialog.js

```diff
@@ -262,10 +262,10 @@ var LinkDialog = Dialog.extend({
         var $url = this.$('input[name="url"]');
         var url = $url.val();
         var label = this.$('input[name="label"]').val() || url;
-
+        label = _.escape(label);
```

Comment:
Maybe even make the `_escape` part of the original declaration / assignment? I don't really see the need for a separate one.